app/services/bike_searcher.rb

```diff
@@ -148,7 +148,7 @@ def fuzzy_find_serial
     bike_ids = fuzzy_find_serial_ids
     # Don't return exact matches
     bike_ids = bike_ids.uniq - matching_serial.map(&:id)
-    Bike.where("id in (?)", bike_ids)
+    Bike.where(id: bike_ids)
```

Comment:
Replacing `id in (?)` with `id: ` here and below resolves SQL generation errors caused by ambiguous table bugs.